scripts/serve-pimcore.sh

```diff
@@ -1,93 +1,155 @@
+        try_files /var/assets\$uri \$uri =404;
+        expires 2w;
+        access_log off;
+        log_not_found off;
+        add_header Cache-Control \"public\";
+    }
+
+    # Installer
+    # Remove this if you don't need the web installer (anymore)
+    if (-f \$document_root/install.php) {
+        rewrite ^/install(/?.*) /install.php last;
+    }
+
+    location / {
+        error_page 404 /meta/404;
+        add_header \"X-UA-Compatible\" \"IE=edge\";
+        try_files \$uri /app.php\$is_args\$args;
+    }
+
+    # Use this location when the installer has to be run
+    # location ~ /(app|install)\.php(/|\$) {
+    #
+    # Use this after initial install is done:
+    location ~ /(app|install)\.php(/|\$) {
+        send_timeout 1800;
+        fastcgi_read_timeout 1800;
+        # regex to split \$uri to \$fastcgi_script_name and \$fastcgi_path
+        fastcgi_split_path_info ^(.+\.php)(/.+)\$;
+        # Check that the PHP script exists before passing it
+        try_files \$fastcgi_script_name =404;
+        include fastcgi.conf;
+        # Bypass the fact that try_files resets \$fastcgi_path_info
+        # see: http://trac.nginx.org/nginx/ticket/321
+        set \$path_info \$fastcgi_path_info;
+        fastcgi_param PATH_INFO \$path_info;
+
+        # Activate these, if using Symlinks and opcache
+        # fastcgi_param SCRIPT_FILENAME \$realpath_root\$fastcgi_script_name;
+        # fastcgi_param DOCUMENT_ROOT \$realpath_root;
+
+        fastcgi_pass php-pimcore5;
+        # Prevents URIs that include the front controller. This will 404:
+        # http://domain.tld/app.php/some-path
+        # Remove the internal directive to allow URIs like this
+        internal;
+    }
+
+    # PHP-FPM Status and Ping
+    location /fpm- {
+        access_log off;
+        include fastcgi_params;
+        location /fpm-status {
+            allow 127.0.0.1;
+            # add additional IP's or Ranges
+            deny all;
+            fastcgi_pass php-pimcore5;
+        }
+        location /fpm-ping {
+            fastcgi_pass php-pimcore5;
+        }
+    }
+    # nginx Status
+    # see: https://nginx.org/en/docs/http/ngx_http_stub_status_module.html
+    location /nginx-status {
+        allow 127.0.0.1;
+        deny all;
+        access_log off;
+        stub_status;
+    }
+
+    ssl_certificate     /etc/nginx/ssl/$1.crt;
+    ssl_certificate_key /etc/nginx/ssl/$1.key;
+}
 "
 
-echo "$block" > "/etc/apache2/sites-available/$1.conf"
-echo "$blockssl" > "/etc/apache2/sites-available/$1-ssl.conf"
-
-sudo a2dissite 000-default
-sudo a2ensite $1 $1-ssl
-
-ps auxw | grep apache2 | grep -v grep > /dev/null
+echo "$block" > "/etc/nginx/sites-available/$1"
+ln -fs "/etc/nginx/sites-available/$1" "/etc/nginx/sites-enabled/$1"
 
-sudo a2enmod ssl rewrite proxy proxy_fcgi
-service apache2 restart
-
-if [ $? == 0 ]
-then
-    service apache2 reload
-fi
+sudo service nginx start
```

Comment:
This should be a restart: `sudo service nginx restart`

Reply:
I didn't use restart because I'm stopping the service at the beginning of the script (line 4).
Do you still want me to change it to restart?

Reply:
You can remove the nginx stop at the top of the script. since we're not using apache we can assume nginx is running and will want to restart it at the end.